### DISCUSSION TITLE
Data sources completeness

### DISCUSSION THREAD

**Discussion by danieldegroot2** [xspanger3770/GlobalQuake#76] [General]
Do you currently have the data sources you need, or are you still looking for more data sources?

**Comment by YacineBoussoufa** [comment]
All the current Data Sources are those included here: https://github.com/YacineBoussoufa/EarthquakeDataCenters
There are more sources available but at the moment I cannot share them

**Comment by danieldegroot2** [comment]
Ok, thanks for sharing.
